packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -1,31 +1,25 @@
 ---
-component: Reference
-title: API Reference
-top_links:
-  - name: API Concepts
-    path: /docs/reference/api-overview/
-    icon: icons/icon--docs-apiconcepts.svg
-  - name: Error Codes
-    path: /docs/reference/error-codes/
-    icon: icons/icon--docs-errorcodes.svg
-  - name: Okta Expression Language
-    path: /docs/reference/okta-expression-language/
-    icon: icons/icon--docs-expressionlang.svg
-  - name: Postman Collections
-    path: /docs/reference/postman-collections/
-    icon: icons/icon--docs-postman.svg
+title: Reference overview
 ---
 
-::: slot left
+# Reference overview
+
+Details on parameters, requests, and responses for Okta's API endpoints.
+
+
+Okta APIs generally fall two categories: signing in your users, and managing
+resources in your Okta org.
+
 ## Sign in Your Users
 API endpoints to authenticate your users, challenge for factors, recover passwords, and more. [Learn about which APIs to use.](/docs/concepts/oauth-openid/#authentication-api-vs-oauth-20-vs-openid-connect)
+API endpoints to authenticate your users, challenge for factors, recover passwords, and more.
```

Comment:
API endpoints ~to~ authenticate your users, challenge for factors, recover passwords, and more.

(Clarifies what the API endpoints can do)

Reply:
done